docs/src/benchmarks/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Benchmarks
+sidebar_position: 10
+---
+
+# Benchmarks
```

Comment:
A key point is missing: what is the workload used for these benchmarks?

Reply:
Agree fully, I put it in the ToDo section as decision needed what we want to be the workload